src/main/java/com/neonlab/product/entities/Driver.java

```diff
@@ -0,0 +1,28 @@
+package com.neonlab.product.entities;
+
+import com.neonlab.common.entities.Generic;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Table(name = "driver")
+public class Driver extends Generic {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private String driverId;
```

Comment:
Rename from `DriverId` to `id`. As the variable will always be used like Driver.getId() this communicates that we are having the drivers Id.

cc- @shrawanindeshpande